src/python/grpcio/tests/interop/client.py

```diff
@@ -109,8 +104,7 @@ def _test_case_from_arg(test_case_arg):
   for test_case in methods.TestCase:
     if test_case_arg == test_case.value:
       return test_case
-  else:
```

Comment:
glint complained:
W0120:107:_test_case_from_arg: Else clause on loop without a break statement [useless-else-on-loop]

I've changed it back though.